persist-cli/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -118,6 +125,9 @@ public void execute() {
             errStream.println("ERROR: the persist directory does not contain any model definition file. " +
                     "run `bal persist init` to initiate the project before generation.");
             return;
+        } else if (schemaFilePaths.size() > 1) {
+            errStream.println("ERROR: persist directory contains multiple schema files. ");
```

Comment:
```suggestion
            errStream.println("ERROR: the persist directory allows only one model definition file, but contains many files.");
```

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)